<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationConf</class>
 <widget class="QWidget" name="LocationConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {font-weight:bold;}</string>
       </property>
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="geolocationRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Geolocation Dataengine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="manualRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Manual</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="PlaceRadioButton">
            <property name="text">
             <string>By Place:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Country:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>City:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="countryLineEdit"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="cityLineEdit"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Country Code:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="countryCodeLineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="addressLabel">
              <property name="text">
               <string>Address:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="addressLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="coordinatesRadioButton">
            <property name="text">
             <string>By Coordinates:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="latitudeLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="longitudeLineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="checkPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Check</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">QLabel {font-weight:bold;}</string>
       </property>
       <property name="text">
        <string>Range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rangeSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Radius around your location for plugins which support it.</string>
       </property>
       <property name="suffix">
        <string>KM</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>geolocationRadioButton</tabstop>
  <tabstop>manualRadioButton</tabstop>
  <tabstop>PlaceRadioButton</tabstop>
  <tabstop>countryCodeLineEdit</tabstop>
  <tabstop>countryLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>coordinatesRadioButton</tabstop>
  <tabstop>latitudeLineEdit</tabstop>
  <tabstop>longitudeLineEdit</tabstop>
  <tabstop>checkPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>manualRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>countryCodeLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>countryLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cityLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>latitudeLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>longitudeLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkPushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>addressLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
